src/language-markdown/print-whitespace.js

```diff
@@ -104,7 +104,7 @@ function isInSentenceWithCJSpaces(path) {
  * @param {AstPath} path path of given node
  * @param {WhitespaceValue} value value of given node (typically `" "` or `"\n"`)
  * @param {AdjacentNodes | undefined} adjacentNodes adjacent sibling nodes of given node
- * @returns {boolean} `true` if given node can be converted to space, `false` if not (i.e. newline or empty character)
+ * @returns {boolean} `true` if given node can be convertedIoSpace, `false` if not (i.e. newline or empty character)
  */
 function canBeConvertedToSpace(path, value, adjacentNodes) {
   // "\n" or " ", of course " " always can be converted to Space
```

Comment:
Is this function ever called for values that aren't `\n`?

Reply:
No. I will remove it from arguments.